tasks/system_probe.py

```diff
@@ -1535,3 +1535,12 @@ def test_microvms(
     ctx.run(
         f"cd ./test/new-e2e && go run ./scenarios/system-probe/main.go --name usama-saqib-test {go_args} --shutdown-period 720",
     )
+
+
+@task
+def generate_git_file_diff(ctx):
```

Comment:
This seems like a generic task that could be reused in other contexts, any reason why this should be put in the `system-probe` namespace?

Reply:
No real reason apart from the fact that it's only used by system probe yet, I'll move it